samples/bluetooth/peripheral_hids_mouse/src/main.c

```diff
@@ -678,6 +681,7 @@ void button_changed(uint32_t button_state, uint32_t has_changed)
 
 	memset(&pos, 0, sizeof(struct mouse_pos));
 
+#if defined(CONFIG_BT_HIDS_SECURITY_ENABLED)
```

Comment:
You can you here IS_ENABLED() macro and then there won't be need to have conditional compilation in 121 line